ODPi-Runtime.md

```diff
@@ -120,6 +121,7 @@ Minimum Versions
 
 Applications on Unix platforms need to understand the base specification of some key components of which they write software. Two of those components are the Java runtime environment and the shell environment.
 
+-   **[TEST_ENVIRONMENT]** **OS:** ODPi Platforms SHOULD be clear what operating systems are supported.
```

Comment:
I don't think it is. This came from IBM's team and I feel it is required as part of HCFS. Further questions should be addressed to @lcham